frontend/src/components/common/Sidebar.vue

```diff
@@ -0,0 +1,90 @@
+<template>
```

Comment:
Let's rename this file to NavBar, it's a little bit confused compare with the ProfileSidebar.

Reply:
Ok, this will be renamed.